fili-core/src/main/java/com/yahoo/bard/webservice/config/FeatureFlag.java

```diff
@@ -6,6 +6,9 @@
  * Feature flags bind an object to a system configuration name.
  */
 public interface FeatureFlag {
+
+    SystemConfig SYSTEM_CONFIG = SystemConfigProvider.getInstance();
```

Comment:
No need to define a variable here maybe? I would personally prefer put it in the default method. And if we do happen to choose a not so common practice of putting a variable here, it should be accessible by all that implement this interface and therefore no need to define their own `SYSTEM_CONFIG` it seems.